src/main/java/org/isf/opd/gui/OpdEditExtended.java

```diff
@@ -567,7 +578,7 @@ private void initialize() {
 		this.setContentPane(getMainPanel());
 		pack();
 		setMinimumSize(this.getSize());
-		setLocationRelativeTo(null);
```

Comment:
please revert this deletion

setLocationRelativeTo(null); makes the window to appear screen centered.